doc/radosgw/layout.rst

```diff
@@ -21,32 +21,37 @@ Although RADOS only knows about pools and objects with their xattrs and
 omap[1], conceptually RGW organizes its data into three different kinds:
 metadata, bucket index, and data.
 
-* Metadata
+Metadata
+^^^^^^^^
 
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
 You can use the following commands to introspect metadata entries:
 
-$ radosgw-admin metadata list
+.. code:: bash
```

Comment:
this is not BASH though. Simply by indenting after `::` will format this correctly